packages/datepickers/src/styled/styled-datepicker.tsx

```diff
@@ -184,10 +211,10 @@ export const StyledDay = styled.div<IStyledDayProps>`
   justify-content: center;
   align-items: center;
   border-radius: 50%;
-  font-size: ${props => (props.small ? DEFAULT_THEME.fontSizes.sm : DEFAULT_THEME.fontSizes.md)};
+  font-size: ${props => (props.small ? props.theme.fontSizes.sm : props.theme.fontSizes.md)};
```

Comment:
Shouldn't this be `props.isCompact` (formerly known as `isSmall`)?

Reply:
This was left as `small` to align with the public prop. I have renamed it to `isSmall` to help differentiate. I will rename it to `isCompact` based on our discussion above.